spring-integration-jmx/src/main/java/org/springframework/integration/jmx/config/MBeanExporterHelper.java

```diff
@@ -84,16 +71,8 @@ public void afterPropertiesSet() throws Exception {
 					className = ((StandardMethodMetadata) def.getSource()).getIntrospectedMethod().getReturnType().getName();
 				}
 				if (StringUtils.hasText(className)){
-					if (className.startsWith(SI_ROOT_PACKAGE)
-							&& !(className.endsWith(IntegrationMBeanExporter.class.getName()))){
-						this.siBeanNames.add(beanName);
-					}
-					if (className.startsWith(SI_ROOT_PACKAGE + "channel.")
-							&& !(className.endsWith(IntegrationMBeanExporter.class.getName()))){
-						this.siBeansToExcludeWhenNoIMBE.add(beanName);
-					}
-					else if (className.equals(IntegrationMBeanExporter.class.getName())) {
-						this.hasIntegrationExporter = true;
+					if (className.startsWith(SI_ROOT_PACKAGE) && !(className.endsWith(IntegrationMBeanExporter.class.getName()))){
```

Comment:
Looks like we can use here `IntegrationConfigUtils.BASE_PACKAGE` and remove both constants in this class.